Neos.EventSourcedContentRepository/Tests/Behavior/Features/EventSourced/Projection/WithDimension.feature

```diff
@@ -3,82 +3,75 @@ Feature: Reading of our Graph Projection
-      | contentStreamIdentifier   | "c75ae6a2-7254-4d42-a31b-a629e264069d"        |
-      | nodeAggregateIdentifier   | "35411439-94d1-4bd4-8fac-0646856c6a1f"        |
+      | contentStreamIdentifier   | "cs-identifier"                               |
+      | nodeAggregateIdentifier   | "nody-mc-nodeface"                            |
       | originDimensionSpacePoint | {"language": "de"}                            |
       | propertyName              | "test"                                        |
       | value                     | {"value": "original value", "type": "string"} |
+    And the graph projection is fully up to date
 
-    When the graph projection is fully up to date
-    And I am in content stream "c75ae6a2-7254-4d42-a31b-a629e264069d" and Dimension Space Point {"language": "mul"}
+    Then I expect the graph projection to consist of exactly 2 nodes
+    And I expect a node with identifier {"nodeAggregateIdentifier": "sir-david-nodenborough", "contentStreamIdentifier": "cs-identifier", "originDimensionSpacePoint": {}} to exist in the content graph
+    And I expect a node with identifier {"nodeAggregateIdentifier": "nody-mc-nodeface", "contentStreamIdentifier": "cs-identifier", "originDimensionSpacePoint": {"language": "de"}} to exist in the content graph
 
-    Then I expect a node "75106e9a-7dfb-4b48-8b7a-3c4ab2546b81" to exist in the graph projection
-    And I expect the node aggregate "00000000-0000-0000-0000-000000000000" to have the following child nodes:
-      | Name | NodeIdentifier                       |
-      | foo  | 75106e9a-7dfb-4b48-8b7a-3c4ab2546b81 |
+    When I am in content stream "cs-identifier" and Dimension Space Point {"language": "gsw"}
+    Then I expect the subgraph projection to consist of exactly 2 nodes
+    Then I expect node aggregate identifier "nody-mc-nodeface" and path "foo" to lead to node {"nodeAggregateIdentifier": "nody-mc-nodeface", "contentStreamIdentifier": "cs-identifier", "originDimensionSpacePoint": {"language": "de"}}
 
-  Scenario: Translation of node in aggregate
+    # check whether we want to test this here at all once node peer variants are implemented
+  #Scenario: Translation of node in aggregate
+  #  When the event NodeAggregateWithNodeWasCreated was published with payload:
+  #    | Key                           | Value                                     |
+  #    | contentStreamIdentifier       | "cs-identifier"                           |
+  #    | nodeAggregateIdentifier       | "nody-mc-nodeface"                        |
+  #    | nodeTypeName                  | "Neos.ContentRepository.Testing:Document" |
+  #    | originDimensionSpacePoint     | {"language": "de"}                        |
+  #    | visibleInDimensionSpacePoints | [{"language": "de"},{"language": "gsw"}]  |
+  #    | parentNodeAggregateIdentifier | "sir-david-nodenborough"                  |
+  #    | nodeName                      | "foo"                                     |#
```

Comment:
quite some stuff commented out here; but I think this is covered by other tests??

Reply:
covered by other tests